repo2docker/buildpacks/conda/environment.frozen.yml

```diff
@@ -1,87 +1,86 @@
-# AUTO GENERATED FROM environment.py-3.7.yml, DO NOT MANUALLY MODIFY
-# Frozen on 2019-02-11 20:26:36 UTC
+# AUTO GENERATED FROM /Users/thead/git/jupyterhub/repo2docker/repo2docker/buildpacks/conda/environment.py-3.7.yml, DO NOT MANUALLY MODIFY
```

Comment:
why did the custom path to your local file get added here?

Reply:
This is another way the "paths got broken" issue raises its ugly head. I'll fix this.